### DISCUSSION TITLE
mcmap v3.0.2 - GUI update

### DISCUSSION THREAD

**Discussion by spoutn1k** [spoutn1k/mcmap#88] [General]
Overhaul of the UI to allow for a more complete experience.
- Updated logging with the use of spdlog;
- Added support for circular coordinates on GUI;
- Added support for tiled output on GUI;
- Added access to memory options on GUI;
- Added colors for the upcoming 1.19 release.

<hr /><em>This discussion was created from the release <a href='https://github.com/spoutn1k/mcmap/releases/tag/'>mcmap v3.0.2 - GUI update</a>.</em>

**Comment by DrakorthaP** [comment]
Hello! Is there any chance of a feature being added like mcmapDZ that generates a silky-smooth, low-bandwidth map that's viewable using a browser?

**Reply by spoutn1k** [reply]
Hey ! I did implement a leaflet solution that allows you to view split up maps using a browser.

This is done by using the `Tiled map output` on the GUI or `-tile` option on the CLI. The map can then be viewed opening the webpage in `/contrib/leaflet/index.html`, and giving it the `mapinfo.json` file in the output.

That behaviour is what I came up with to restore functionality when I took over. I am not familiar with mcmapDZ, and it looks like the links on the minecraft forum are dead, but I can definitely try to make it happen natively with `mcmap`. Let me know:
- How you want the program to behave;
- How you would like to setup the webpage;
- How you expect the output to be bundled.

Thanks !

**Comment by DrakorthaP** [comment]
Hi! thank you very much for reaching out to me regarding this.

We've been trying to update our map render since 2019! But have been stuck here.. If you could make it work natively, It would be very much appreciated.

I have working versions of mcmapdz on my hard drive that I'll share with you. I'm not sure if these files will reveal everything you need to know, but I hope it's better than nothing at all. https://drive.google.com/file/d/1r2bcofzXaf26ws8oTXYi04DyRUfnzWRq/view?usp=share_link

What we really liked about it was the ability to set map labels, as well as how clean and smooth the map was to navigate on the web page.

As for how I'd like the webpage to be setup and how it should look, take a look at our last map render on our website from 2019. Here is a link; https://qkzskrbbwbkyw3lacbvz9w.on.drv.tw/www.taw.map/

Thank you, and please let me know if anything is missing or if you need anything at all

**Reply by spoutn1k** [reply]
I get an error from google drive on the first link, what happened ?

**Reply by DrakorthaP** [reply]
It seems like google has decided the file type is suspicious and won't allow it to be shared.

Here's a dropbox link;
https://www.dropbox.com/s/qb5u0becvaufdpq/mcmapdz%20for%201.13.zip?dl=0

**Reply by spoutn1k** [reply]
Snap, you would not happen to have the sources, would you ? A quick google search cannot find any worthwhile reference to hdmake, used by the DZ version.

**Reply by DrakorthaP** [reply]
hello,  she no longer has the files I'm sorry to say :(

**Comment by DrakorthaP** [comment]
I don't have the source files. a friend was working with them when trying to find a solution for us, but that was a few years ago now. I doubt she will still have them but I will try to find out.

**Reply by spoutn1k** [reply]
Please outline what you would like / how mcmapDZ worked on https://github.com/spoutn1k/mcmap/discussions/91
